explorer/k8s/helm/explorer-squid/templates/postgres-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "explorer-squid.fullname" . }}-postgres-configmap
+  namespace: {{ .Values.namespace }}
+data:
+  postgres.conf: |-
+    max_connections = 200
+    shared_buffers = 8GB
+    effective_cache_size = 24GB
+    maintenance_work_mem = 2GB
+    checkpoint_completion_target = 0.9
+    wal_buffers = 16MB
+    default_statistics_target = 100
+    random_page_cost = 1.1
+    effective_io_concurrency = 200
+    work_mem = 10485kB
+    min_wal_size = 1GB
+    max_wal_size = 4GB
+    max_worker_processes = 16
+    max_parallel_workers_per_gather = 4
+    max_parallel_workers = 16
+    max_parallel_maintenance_workers = 4
+    shared_preload_libraries='pg_stat_statements'
+    pg_stat_statements.track=all
+    # Replace with particular IP addresses in production
+    listen_addresses = 'REPLACE_IP_ADDRESS'
```

Comment:
Where does this gets replaced? I'm not seeing any usage of replace with actual ip anywhere

Reply:
updated the `listen_address` to `0.0.0.0` for now.